extensions/interactions/EndExploration/directives/end-exploration-validation.service.spec.ts

```diff
@@ -132,6 +136,9 @@ describe('EndExplorationValidationService', () => {
 
   it('should catch non-string value for recommended exploration ID',
     () => {
+      // Ts ignore is used because here knowingly we are assigning the
+      // wrong type of value to test warnings.
+      // @ts-ignore
```

Comment:
```suggestion
      // TS ignore is used here because we are assigning the wrong type of 
      // value to test the warnings.
      // @ts-ignore
```

Reply:
ditto below

Reply:
done